data_model/src/asset.rs

```diff
@@ -482,7 +482,7 @@ impl FromStr for AssetId {
         if let Some((asset_definition_candidate, account_id_candidate)) = string.rsplit_once('#') {
             let account_id: AccountId = account_id_candidate.parse()
                 .map_err(|_err| ParseError {
-                    reason: "Failed to parse the `account_id` part of the `asset_id`. Please ensure that it has the form `account@domain`"
+                    reason: "Failed to parse the `account_id` part of the `asset_id`. Please ensure that it has the form `asset##account@domain`"
```

Comment:
I don't think this change is valid since it talks about `account_id` part of `asset_id`:
`asset##[accound@account_domain]`, `asset#asset_domain#[account@account_domain]`.

Reply:
I remember that I had to change it becuse I had a very misleading error. I will see if I could replicate it if not I will revert it back